govcd/externalnetwork_test.go

```diff
@@ -0,0 +1,143 @@
+// +build extnetwork network functional ALL
+
+/*
+ * Copyright 2019 VMware, Inc.  All rights reserved.  Licensed under the Apache v2 License.
+ */
+
+package govcd
+
+import (
+	"fmt"
+	"github.com/vmware/go-vcloud-director/v2/types/v56"
+	. "gopkg.in/check.v1"
+	"net/url"
+	"time"
+)
+
+// Retrieves an external network and checks that its contents are filled as expected
+func (vcd *TestVCD) Test_ExternalNetworkGetByName(check *C) {
+	fmt.Printf("Running: %s\n", check.TestName())
+	if vcd.skipAdminTests {
+		check.Skip(fmt.Sprintf(TestRequiresSysAdminPrivileges, check.TestName()))
+	}
+
+	externalNetwork, err := GetExternalNetwork(vcd.client, vcd.config.VCD.ExternalNetwork)
+	check.Assert(err, IsNil)
+
+	check.Assert(externalNetwork.ExternalNetwork.Name, Equals, vcd.config.VCD.ExternalNetwork)
+}
+
+// Tests System function Delete by creating external network and
+// deleting it after.
+func (vcd *TestVCD) Test_ExternalNetworkDelete(check *C) {
+	fmt.Printf("Running: %s\n", check.TestName())
+	if vcd.skipAdminTests {
+		check.Skip(fmt.Sprintf(TestRequiresSysAdminPrivileges, check.TestName()))
+	}
+
+	if vcd.config.VCD.ExternalNetwork == "" {
+		check.Skip("Test_GetByName: External network isn't configured. Test can't proceed")
```

Comment:
Unless this is some special construct I don't know:
```suggestion
		check.Skip("Test_ExternalNetworkDelete: External network isn't configured. Test can't proceed")
```

Reply:
fixed